<!-- Joystick Control AXON platform -->
<launch>
  <param name="/use_sim_time" value="false"/>

  <node pkg="axon_link" name="axon_link_vonly" type="axon_link_vonly" output="screen">
	<rosparam file="$(find axon_link)/config/axon_param.yaml" command="load"/>        
  </node>

  <node pkg="joy" name="joy_node" type="joy_node">
    <param name="joy_node/dev" value="/dev/input/js0"/>
  </node>

  <node pkg="axon_link" name="joy_ctrl_axon_node" type="joy_ctrl_axon_node">
  	<remap from="cmd_vel" to="axon_cmd_vel"/>
  </node>

  <!-- Record SLAM Pos., IMU, VICON Pos. and plot SLAM path -->
  <include file="$(find slam2d)/launch/slam2d.launch">
        <arg name="log_file_save_flag" value="true"/>
	<arg name="need_virual_tf_base_link" value="true"/>
    	<arg name="need_virual_tf_target" value="true"/>
	<arg name="use_vicon_pose" value="true"/>
        <arg name="robot_velocity_name" value="/axon_cmd_vel"/>
    	<arg name="motor_velocity_name" value="/axon_wlr_cmd"/>
	<arg name="node_info_display" value="log"/>
  </include>
</launch>
